backend/app/manage.go

```diff
@@ -13,6 +13,11 @@ import (
 
 // ManageOpts can modify Manage behaviour.
 type ManageOpts struct {
+	// ApiVersion is the required ApiVersion -- setting this will
+	// fail any request asking for a different version.  For multi-version servers,
+	// this value should not be set, and be handled directly in each server
+	APIVersion string
```

Comment:
Why shouldn't the SDK support multi-version and/or why would you want to populate this? Hard to understand without seeing the full picture.

Reply:
Good question -- I'll remove this config and make the api-management entirely up to the handler.